mlrun/frameworks/_ml_common/pkl_model_server.py

```diff
@@ -56,3 +58,13 @@ def predict(self, request: dict) -> list:
         y_pred: np.ndarray = self.model.predict(x)
 
         return y_pred.tolist()
+
+    def explain(self, request: Dict[str, Any]) -> str:
+        """
+        Return a string explaining what model is being serve in this serving function and the function name.
+
+        :param request: A given request.
+
+        :return: Explanation string.
+        """
+        return f"The '{self.model.name}' model serving function named '{self.name}'"
```

Comment:
pickle model server is used for `lightgbm`, `scitki-learn` and `xgboost` which all don't have `name` property in their models. You can simply write something like:

```suggestion
        return f"A model server named '{self.name}'"
```

Reply:
@guy1992l - is there a point adding UT for coverage? or model could be arbitrary?

Reply:
Not really... we do need to think of a way to test all of these though per framework like I did for training. It does require some work, but there is no need for special unit tests for this class.